patch

```diff
@@ -181,7 +183,46 @@
  import shutil
  import struct
  import sys
-@@ -32,7 +30,8 @@
+@@ -23,6 +21,38 @@
+ 
+ try:
+     import numpy as np
++
++    # This function is used for numpy serialization with pickle5 in Python3.5.
++    # It can be removed once numpy enables Pickle Protocol 5 for Python3.5.
```

Comment:
This doesn't sound necessary. Just don't test Numpy with Python 3.5. Numpy should run their own tests anyway.